ftd/t/js/80-or-type-constant.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    
+    <meta content="fastn" name="generator">
+    
+    
+    <script>
+        let __fastn_package_name__ = "foo";
+    </script>
+
+    <script src="fastn-js.js"></script>
+    
+
+    <style>
+       
+    </style>
+</head>
+<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0">
+<body data-id="1"><div data-id="2" class="ft_column __w-1 __h-2"><div data-id="3">Smol</div><div data-id="4">Smol</div></div></body><style id="styles">
+    .__w-1 { width: 100%; }
+	.__h-2 { height: 100%; }
+    </style>
+<script>
+    (function() {
+        let main = function (parent) {
+  let __fastn_super_package_name__ = __fastn_package_name__;
+  __fastn_package_name__ = "foo";
+  try {
+    let parenti0 = fastn_dom.createKernel(parent, fastn_dom.ElementKind.Text);
+    parenti0.setProperty(fastn_dom.PropertyKind.StringValue, fastn.formula([global.foo__b,
+    global.foo__small], function () {
+      if (function () {
+        return (fastn_utils.getter(global.foo__b) == fastn_utils.getter(global.foo__small));
```

Comment:
```suggestion
        return (fastn_utils.getter(global.foo__b) == fastn_utils.getter(global.foo__button_type.get("small")));
```

Reply:
I see that you have created a new variable `small` using `-- button-type small: small` and you are using it. So I think then it's fine.

Reply:
But I want another example where you'll not create any variable and use the or-type variant directly
```
if: { b == medium }
```
Where `medium` is not a variable